test/README.md

```diff
@@ -85,6 +85,10 @@ To run the script for all end to end test images:
 A docker tag may be passed as an optional parameter. This can be useful on
 Minikube in tandem with the `--tag` [flag](#using-a-docker-tag):
 
+`PLATFORM` environment variable is optional. If it is specified, test images
```

Comment:
Should this be `PLATFORMS` with s since the point is there could be more than one?